src/ActionModal/components/ActionModal.vue

```diff
@@ -435,7 +441,9 @@ export default {
     ...mapGetters({ networkId: `network` }),
     checkFeatureAvailable() {
       const action = `action_` + this.featureFlag
-      return this.network[action] === true
+      return typeof this.network[action] === `string`
```

Comment:
can we flip it and convert the string/null to the enum?

Reply:
```
this.network[action] !== `string` ? networkCapabilityLookup[this.network[action]] : this.network[action]
```

Reply:
This way we already can implement the handling in the future way

Reply:
You mean convert the Boolean to the enum values?

Reply:
Oh, never mind. I didn't see the latter two messages. I think GitHub is not working so properly lately.

Btw, it should be `this.network[action] !== `string` in the line you suggested

Reply:
No! I wanted you to convert a `true` to `ENABLED` and only use `ENABLED` in the UI

Reply:
I hope now I could apply you suggestion correctly. If not, please make a suggestion or add something tangible. It would save up sooo much time

Reply:
fixed the logic and refactored for you

Reply:
Cool, because by what you were saying I could have never guessed what you wanted 🤷‍♀